website/docs/r/cloud_backup_snapshot_restore_job.html.markdown

```diff
@@ -126,9 +126,7 @@ resource "mongodbatlas_cloud_backup_snapshot_restore_job" "test" {
 ```
 
 ### Available complete examples
-- [Restore from automated backup snapshot](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/examples/test-upgrade/v110/cloud-backup-snapshot/v110)
-- [Restore from backup snapshot download](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/examples/test-upgrade/v100/design-id-reference/snapshot-restore)
```

Comment:
are we sure these are outdated? If these are relevant use-cases, could we look into just keeping these as examples?

Reply:
they are 3 links,  link 2 and 3 are broken, I updated link 3.
checking if link 1 is relevant and will move it to examples folder or will use an existing one

Reply:
updated to have 2 meaningful examples